tests/integration/components/donations/create-donation-test.js

```diff
@@ -0,0 +1,84 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import Ember from 'ember';
+import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+
+import createDonationComponent from '../../../pages/components/donations/create-donation';
+
+const {
+  K
+} = Ember;
+
+let page = PageObject.create(createDonationComponent);
+
+function setHandlers(context, { continueHandler = K } = {}) {
```

Comment:
Same as the other test. Not sure the second param here needs to be a hash since we only ever call one property from it.